### PR TITLE
Greps for pronoun helper spelling mistakes

### LINKED ISSUE #83083: "He seem%PRONOUN_S unable to stay still" - examine text
Reporting client version: 515.1637

## Issue Summary
image

## Round ID:
[228404](https://scrubby.melonmesa.com/round/228404)

## Testmerges:
- [Wawastation](https://www.github.com/tgstation/tgstation/pull/82298)
- [Implements dynamic scaling for spies](https://www.github.com/tgstation/tgstation/pull/82961)
- [Fixes lua breaking when registering signals on turfs](https://www.github.com/tgstation/tgstation/pull/83018)
- [[testmerge] Split up a key parallax proc for profiler purposes](https://www.github.com/tgstation/tgstation/pull/83031)

## Reproduction:
I have no idea, I just examined the guy and got this. Probably something wrong with the file for that text

### PR DESCRIPTION
**Pull request by ShizCalev**
Fixes #83083

These are case sensitive

### CODE REVIEW

File: code/__DEFINES/pronouns.dm [hunk 1]
```diff
@@ -17,6 +17,8 @@
 /// s, null (she looks, they look) "%PRONOUN_s" = "p_s"
 /// es, null (she goes, they go) "%PRONOUN_es" = "p_es"
 
+/// 	Don't forget to update the grep if you add more!
```
Comment (optimumtact) [inline code/__DEFINES/pronouns.dm:20]: add a path maybe

**Comment by ZephyrTFA** [comment]
check grep is deprecated because it's stupid.
there is currently no alternative
but I'm just letting you know

**Review by ZephyrTFA — APPROVED** [review]
approving changes to check_grep.sh

**Comment by 00-Steven** [comment]
baller

**Pull request merged**